### PR TITLE
Updated Getting Started with grammar and feature changes.

### PR DESCRIPTION
**Pull request by zardra**

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @matthewjohnston4 to be potential reviewers.

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: docs/_docs/quick-start/getting-started.md [hunk 1]
```diff
@@ -25,54 +24,53 @@ apm install nuclide
 > Nuclide is [not currently supported on Windows](/docs/editor/setup/#windows).
 
 While technically optional, in order for features such as [`Quick Open`](#quick-open) to work
-correctly, you will also need to install [`Watchman`](https://facebook.github.io/watchman/) and
-ensure it is on your `$PATH`. There are other
-[recommended installations](/docs/editor/setup/#post-installation) as well.
+correctly, you also need to install [`Watchman`](https://facebook.github.io/watchman/) and
+ensure it is in your `$PATH` enviroment variable. There are other
```
Comment (JoelMarcey) [inline docs/_docs/quick-start/getting-started.md]: spelling - s/enviroment/environment

File: docs/_docs/quick-start/getting-started.md [hunk 2]
```diff
@@ -25,54 +24,53 @@ apm install nuclide
...
+- The left side-pane is the Nuclide File Tree view.
+- The main pane contains introductory information about Nuclide and the Quick Launch Menu. This is also where you will edit your files (just like in normal Atom).
+- The bottom status bar shows you error and health statistics.
 
 ## Adding a Project
 
-After launching Nuclide, the next common step is to open a project that you would like to work on.
-This could be a Hack, Flow or any other project that has a root directory.
+The first common step after launching Nuclide is to open a project you would like to work on.
+This could be a Hack, Flow, or any other project that has a root directory.
```
Comment (JoelMarcey) [inline docs/_docs/quick-start/getting-started.md]: Probably could link to the actual Hack and Flow pages that we have at nuclide.io. But not a stopper for this pull request, unless you want to when making other updates.

File: docs/_docs/quick-start/getting-started.md [hunk 3]
```diff
@@ -25,54 +24,53 @@ apm install nuclide
...
 
 ![](/static/images/docs/quick-start-getting-started-quick-launch.png)
 
 ## Quick Open
 
-Nuclide's [*Quick Open*](/docs/features/quick-open) feature gives you access to Nuclide's file
-search mechanism, which includes *omnisearch* which can find anything in your opened project.
-Click on `Try It`, or `Cmd-T` (`ctrl-T` on Linux) to access the feature.
+The [*Quick Open*](/docs/features/quick-open) feature gives you access to Nuclide's file
+search mechanism, including *OmniSearch* which can find any files in your opened project and show you which have been recently opened.
```
Comment (JoelMarcey) [inline docs/_docs/quick-start/getting-started.md]: So, in theory we can find stuff in files.... but it depends on the type of project you are working on. Maybe we could say something like:

> , including *OmniSearch, which quickly displays recently opened files, quick searches for files based on partial names and, depending, on the project, can search within files for symbols, etc.

>

File: docs/_docs/quick-start/getting-started.md [hunk 4]
```diff
@@ -25,54 +24,53 @@ apm install nuclide
...
+Click `Try It` or use the `Cmd-T` keyboard shortcut (`Ctrl-T` on Linux) to access the feature.
 
 ![](/static/images/docs/quick-start-getting-started-quick-open.png)
 
-You can also search directly by filename, content within open files and see which files have been
+You can also search filenames in your project, filenames of currently open files, and see which files have been
```
Comment (JoelMarcey) [inline docs/_docs/quick-start/getting-started.md]: Should we add "by" like "You can also search by filenames in your project...."?

**Comment by JoelMarcey** [comment]
Just a few things to update. But general improvement. Thanks!

**Comment by facebook-github-bot-6** [comment]
@zardra updated the pull request - [view changes](https://github.com/facebook/nuclide/pull/723/files/4dc840bb3965143b93f73ede1173e3345906b26b..024f93c9c9ab7714fcc432c6cbb082a0d5210351)

**Pull request closed without merging**